website/draft_site/drafts/templates/drafts/show_pack.html

```diff
@@ -26,6 +29,13 @@
         <img src="{{ image_url }}" width="146" height="204" />
     {% endfor %}
 {% endif %}
+    <input type="button" onclick="unhide(this, 'bot_ratings') " value="Show Bot Ratings">
+
+    <div id="bot_ratings" class="hidden">
+        {% for rating in bot_ratings %}
+        <div >{{ rating.card.name }} : {{ rating.rating }}</div>
```

Comment:
Wouldn't it be better to show the entire Rating object for more info?

Reply:
I had it this way at first, but the rating object is huge, and it obscures the thing you really want.

![image](https://user-images.githubusercontent.com/17679150/60778965-d1db1500-a0ed-11e9-93ad-d30ead5601a7.png)
It's pretty difficult to read, imo.

Reply:
We could certainly put the data in tabular format, which would probably clean it up a fair bit. That would mean the tags would all occupy one column, though, which would not be great.

Reply:
The majority of the clutter comes from the full Card object being printed. You could do something like this to replace that with the card name: https://github.com/uponthesun/mtg-draft-ai/blob/master/mtg_draft_ai/brains.py#L152

Debatably we should make the name-only printing the default way to print out the object by changing/removing the __repr__ implementation on the Card object: https://github.com/uponthesun/mtg-draft-ai/blob/master/mtg_draft_ai/api.py#L34 I would be okay with that too.